source/core.py

```diff
@@ -898,10 +898,11 @@ def main():
 	log.debug("Initializing global plugin handler")
 	globalPluginHandler.initialize()
 
-	log.debug("Initializing remote client")
 	import remoteClient
 
-	remoteClient.initialize()
+	if config.conf["remote"]["enabled"]:
```

Comment:
Also here, rather than checking the config in core, keep it up to the remote client itself to discard the init call when remote is disabled. This also ensures that a malicious actor can't circumvent the config check and call initialize on the remote client.